### PR TITLE
More fixes

### PR DESCRIPTION
**Pull request by EhsanKia**
Bunch more fixes

- Make /join and /part work along side anonchat setting
- Fix BTTV spammers loading from API
- Added "/followed" command to get follow age
- Fix svg for leaving group chat not showing up
- Fix BVL icon not being centered
- Add chatters count to BVL

### CODE REVIEW

File: src/js/features/better-viewer-list.js [hunk 1]
```diff
@@ -21,12 +22,18 @@ function renderViewerList() {
...
             if (results[i].filter && results[i + 1].text === ' ') i++;
             else tmpResults.push(results[i]);
         }
         results = tmpResults;
+    } else {
+        count = chatterCount;
     }
     viewList.render(results);
+
+    var label = count + (count < 2 ? ' viewer' : ' viewers');
```
Comment (night) [inline src/js/features/better-viewer-list.js]: 1. Your counting system seems pointless. Why not just `results.length`?
2. "chatters" not "viewers"

File: src/js/chat/helpers.js [hunk 2]
```diff
@@ -977,6 +977,16 @@ exports.translate = function($element, sender, text) {
     });
 };
 
+exports.followDate = function(user, channel) {
+    bttv.TwitchAPI.get('users/' + user + '/follows/channels/' + channel).done(function(data) {
+        var m = moment(data.created_at);
+        reply = 'Followed ' + m.fromNow() + ' (' + m.format('LLL') + ')';
```
Comment (night) [inline src/js/chat/helpers.js]: missing `var` ?

also maybe make the response more verbose, include the channel name for clarity, maybe accept specifying a channel as a param, default to current channel

File: src/js/features/better-viewer-list.js (hunk 1 above)
Comment (EhsanKia) [inline src/js/features/better-viewer-list.js]: The problem is that the list includes headers such as "MODERATORS" or "ADMINS" and I can't predict how many of those there are in the list.

The chatter/viewer thing really bugs me too, the issue is, it's called "viewers list" even though it's supposed to be "chatters list", but yeah I'm stuck between changing everything to chatter, or having it be inconsistent. Both solutions kinda suck.

Comment (night) [inline src/js/features/better-viewer-list.js]: Why are those in the list?

**Comment by EhsanKia** [comment]
That's just the data structure I'm using. Trust me it's the cleanest way to do it. It's a linear pass that's happening anyway to filter so there's no extra cost to count anyway.

**Pull request merged**